downstream/modules/platform/proc-settings-gw-additional-options.adoc

```diff
@@ -0,0 +1,32 @@
+[id="proc-settings-gw-other-options"]
+
+= Configuring additional platform options
+
+//Content divided into multiple procedures to address issue AAP-30592
+
+From the *Platform gateway settings* page, you can configure additional platform options.
+
+.Procedure
+. From the navigation panel, select {MenuSetGateway}.
+. The *Platform gateway settings* page is displayed. 
+. Click btn:[Edit platform gateway settings].
+. You can configure the following *Other settings*:
++
+* *JWT expiration buffer in seconds*: The number of seconds before a JWT token's expiration to revoke from the cache.
++
+When authentication happens a JWT token is created for the user and that token is cached. 
+When subsequent calls happen to services such as automation controller or EDA, the token is taken from the cache and sent to the service. 
+Both the token and the cache of the token have an expiration time. 
+If the token expires while in the cache the authentication process attempts results in a 401 error (unauthorized). 
+This setting gives {PlatformName} a buffer by removing the JWT token from the cache before the token expires. 
+When a token is revoked from cache a new token with a new expiration is generated and cached for the user. 
+As a result, expired tokens from the cache are never used. 
+This setting defaults to 2 seconds. 
+If you have a large latency between platform gateway and your services and observe 401 responses you m  increase this setting to lower the number of 401 responses.
```

Comment:
a stray "m" in this sentence

Reply:
I think this is supposed to be you must; I must have clipped it. Fixed.